cmd/blockchaincmd/deploy.go

```diff
@@ -684,62 +686,59 @@ func deployBlockchain(cmd *cobra.Command, args []string) error {
 			return err
 		}
 
-		if false {
-			chainSpec := contract.ChainSpec{
+		clusterName, err := getClusterNameFromList()
+		if err != nil {
+			return err
+		}
+
+		if err = node.SyncSubnet(app, clusterName, blockchainName, true, nil); err != nil {
+			return err
+		}
+
+		if err := node.WaitForHealthyCluster(app, clusterName, node.HealthCheckTimeout, node.HealthCheckPoolTime); err != nil {
+			return err
+		}
+
+		chainSpec := contract.ChainSpec{
+			BlockchainName: blockchainName,
+		}
+		_, genesisPrivateKey, err := contract.GetEVMSubnetPrefundedKey(
+			app,
+			network,
+			chainSpec,
+		)
+		if err != nil {
+			return err
+		}
+		rpcURL, _, err := contract.GetBlockchainEndpoints(
+			app,
+			network,
+			chainSpec,
+			true,
+			false,
+		)
+		if err != nil {
+			return err
+		}
+		if err := validatormanager.SetupPoA(
+			app,
+			network,
+			rpcURL,
+			contract.ChainSpec{
 				BlockchainName: blockchainName,
-			}
-			genesisAddress, genesisPrivateKey, err := contract.GetEVMSubnetPrefundedKey(
-				app,
-				network,
-				chainSpec,
-			)
-			if err != nil {
-				return err
-			}
-			privateKey, err := privateKeyFlags.GetPrivateKey(app, genesisPrivateKey)
-			if err != nil {
-				return err
-			}
-			if privateKey == "" {
-				privateKey, err = prompts.PromptPrivateKey(
-					app.Prompt,
-					"Which key to you want to use to pay for initializing Validator Manager contract? (Uses Blockchain gas token)",
-					app.GetKeyDir(),
-					app.GetKey,
-					genesisAddress,
-					genesisPrivateKey,
-				)
-				if err != nil {
-					return err
-				}
-			}
-			rpcURL, _, err := contract.GetBlockchainEndpoints(
-				app,
-				network,
-				chainSpec,
-				true,
-				false,
-			)
-			if err != nil {
-				return err
-			}
-			if err := validatormanager.SetupPoA(
-				app,
-				network,
-				rpcURL,
-				contract.ChainSpec{
-					BlockchainName: blockchainName,
-				},
-				privateKey,
-				common.HexToAddress(sidecar.PoAValidatorManagerOwner),
-				avaGoBootstrapValidators,
-			); err != nil {
-				return err
-			}
-			ux.Logger.GreenCheckmarkToUser("Subnet is successfully converted into Subnet Only Validator")
+			},
+			genesisPrivateKey,
+			common.HexToAddress(sidecar.PoAValidatorManagerOwner),
+			avaGoBootstrapValidators,
+		); err != nil {
+			return err
+		}
+		ux.Logger.GreenCheckmarkToUser("Subnet is successfully converted into Subnet Only Validator")
+	} else {
+		if err := app.UpdateSidecarNetworks(&sidecar, network, subnetID, blockchainID, "", "", nil); err != nil {
+			return err
```

Comment:
I used reviewable.io which posted the comments above. Suggestion should be: 
```suggestion
		ux.Logger.GreenCheckmarkToUser("L1 is successfully converted to sovereign")
	} else {
		if err := app.UpdateSidecarNetworks(&sidecar, network, subnetID, blockchainID, "", "", nil); err != nil {
			return err
```